carla/models/catalog/load_model.py

```diff
@@ -47,7 +47,10 @@ def load_model(name, dataset, ext="h5", cache=True, models_home=None, **kws):
         )
 
         if not os.path.exists(cache_path):
-            os.makedirs(os.path.dirname(cache_path))
+            len_file_name = len(name + "." + ext)
+            cache_path_without_file = cache_path[:-len_file_name]
+            if not os.path.exists(cache_path_without_file):
+                os.makedirs(os.path.dirname(cache_path_without_file))
```

Comment:
Why did we need to add these lines? What was the problem?

Reply:
It was not possible to load multiple ML models. For example, if someone loads first the tensorflow ann model into cache at '../Users/xxx/carla/models/ann/adult/ann.h5' then it is not possible to load the pytorch model as ann.pt into the same directory.

Conserns #15

Reply:
Adding the parameter worked in my tests 👍